hack/make/release-deb

```diff
@@ -73,7 +73,10 @@ TreeDefault {
 };
 EOF
 
-for suite in $(exec contrib/reprepro/suites.sh); do
+for dir in contrib/builder/deb/${PACKAGE_ARCH}/*/; do
+	version="$(basename "$dir")"
```

Comment:
It's done on a line below too so yeah it works in my shell, and for
uniformity we should use the same or change both

On Tuesday, March 8, 2016, Sebastiaan van Stijn <notifications@github.com>
wrote:

> In hack/make/release-deb
> <https://github.com/docker/docker/pull/21038#discussion_r55417687>:
>
> > @@ -73,7 +73,10 @@ TreeDefault {
> >  };
> >  EOF
> >
> > -for suite in $(exec contrib/reprepro/suites.sh); do
> > +for dir in contrib/builder/deb/${PACKAGE_ARCH}/*/; do
> > +	version="$(basename "$dir")"
>
> oh, hold on, is this good (nested quotes)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/21038/files#r55417687>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>